scripts/updateUID.Dockerfile

```diff
@@ -26,6 +26,9 @@ RUN eval $(sed -n "s/${REMOTE_USER}:[^:]*:\([^:]*\):\([^:]*\):[^:]*:\([^:]*\).*/
 		if [ "$OLD_GID" != "$NEW_GID" ]; then \
 			sed -i -e "s/\([^:]*:[^:]*:\)${OLD_GID}:/\1${NEW_GID}:/" /etc/group; \
 		fi; \
+		if [ ! -d "$HOME_FOLDER" ]; then \
+			mkhomedir_helper ${REMOTE_USER}; \
```

Comment:
This copies `/etc/skel` over only on Linux (the only platform where the UID/GID update runs), but not on Windows or Mac. It is better to keep the platforms aligned. Let's skip the `chown` below when the folder does not exist.

Reply:
oh, ok. I'll send an updated version.

Unrelated but... I love how this dockerfile gets picked up and builds another image (with another name) on top of the user's one. How can I add my own dockerfiles?

I'm changing both the dockerfile and the image name in my own `.devcontainer/docker-compose.yaml` so i can base my `.devcontainer/Dockerfile` on top of my project's own image. It works, but I have to build the regular project image beforehand...

Reply:
Great, thanks!

We currently don't support adding more Dockerfiles beyond this one which we use internally. I suggest you open a feature request. Thanks.